play-media-native/api/queries/getAthletes/index.ts

```diff
@@ -133,7 +133,7 @@ export const getAthleteById = async (id: string): Promise<Athlete> => {
   const athleteResponse = (await fetchGraphQL(getAthleteByIdQuery(id))) as {
     data: { athlete: AthleteResponse };
   };
-  const statusResult = await getItemStatusById(id);
+  const statusResult = await getItemStatusById(id, 'content');
```

Comment:
Use string constants for these strings in constants folder.

`export const STATUS_TYPES = {
content: 'content',
media: 'media'
}`